### DISCUSSION TITLE
Attributify mode with computed values

### DISCUSSION THREAD

**Discussion by Talha-T** [unocss/unocss#2972] [Q&A]
Hello guys,

I really like the idea behind attributify mode, and I think it would be a life-saver if we could use it dynamically like this:

```
const { primary, inverted } = defineProps<Props>();

const bg = primary ? 'fudipu-orange' : inverted ? 'slate-5' : 'default'
const text = primary ? 'white' : inverted ? 'text-gray-8' : 'default'

<button
    :bg="bg" :text="text" un-disabled="text-gray-400 bg-red-100"
/>
```

`un-disabled` tag works because it is fixed value, however it does not work dynamically with `bg` and `text`. Is there any workaround about this issue? It would improve my codebase a ton. 
Thanks.

**Comment by zyyv** [comment]
I suggest you to use full utility class instead of dynamic stitching. This helps uno to statically extract utility in advance.

e.g.
```diff
- const bg = primary ? 'fudipu-orange' : inverted ? 'slate-5' : 'default'
+ const bg = primary ? 'bg-fudipu-orange' : inverted ? 'bg-slate-5' : 'bg-default'
```

```html
<button
    :class="bg" un-disabled="text-gray-400 bg-red-100"
/>
```

Or you can continue to write like this, but only if they may appear in the [`safelist`](https://unocss.dev/guide/extracting#safelist)  in advance.
